test/other/test_emit_tsd.d.ts

```diff
@@ -0,0 +1,7 @@
+interface WasmModule {
+  _fooVoid(): void;
+  _fooInt(_0: number, _1: number): number;
+  _main(_0: number, _1: number): number;
+}
+
+export type MainModule = WasmModule;
```

Comment:
Where does `MainModule` from?

Reply:
Not sure if you mean where the name comes from our where it's defined. MainModule is defined here, it's basically an alias to WasmModule. The name is what embind previously used, so I'm sticking with that for now.

Reply:
Fair enough for now.  I'm not sure why we need alias though.. why not just call it `Module` (or what ever the EXPORT_NAME is?)